src/atomic_store.rs

```diff
@@ -229,15 +233,22 @@ impl AtomicStore {
                 Some(load_info.file_counter)
             },
             resources: load_info.resources,
+            commit_timeout: Duration::from_millis(100),
         })
     }
 
+    /// Set the commit timeout. By default, `AtomicStore` will wait 100 milliseconds for all logs to be committed.
+    pub fn set_commit_timeout(&mut self, timeout: Duration) {
+        self.commit_timeout = timeout;
+    }
+
+    /// Commit the version. Note that *all* `Log` must call `.commit_version()` or `.skip_version()` before this function is called.
```

Comment:
Phrasing is a little awkward. Saying "all logs" may be better, but once the `key_value_store` is added, they won't all be "logs". "all resource stores", maybe. I almost want to create a trait just to have a collective term for the various stores.

Reply:
I think we should replace `Log` by `Storage`. This was something that was confusing me when I started reading up on the source code. So we'd have `AppendStorage`, `FixedAppendStorage` and `RollingStorage`, all part of the `AtomicStore`

Reply:
For now I'll make it `Note that all logs and stores must`